__mocks__/jest-mock.ts

```diff
@@ -74,7 +75,10 @@ jest.doMock('react-native', () => {
         flatten: jest.fn((e) => e)
       },
       Dimensions: {
-        get: () => jest.fn().mockReturnValue({width: 414, height: 818})
+        get: jest.fn().mockReturnValue({width: 300, height: 300})
+      },
+      PixelRatio: {
+        roundToNearestPixel: jest.fn().mockReturnValue(20)
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The mock return values for `Dimensions.get` and `PixelRatio.roundToNearestPixel` have been changed. Ensure these new values are appropriate for the tests that rely on them. If different tests require different values, consider making these mocks more flexible by allowing tests to set their own mock return values.
```diff
-        get: () => jest.fn().mockReturnValue({width: 414, height: 818})
+        get: jest.fn().mockReturnValue({width: 300, height: 300})

-        roundToNearestPixel: jest.fn()
+        roundToNearestPixel: jest.fn().mockReturnValue(20)
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->